examples/AspNetCore/Startup.cs

```diff
@@ -57,7 +57,7 @@ public void ConfigureServices(IServiceCollection services)
             switch (exporter)
             {
                 case "jaeger":
-                    services.AddOpenTelemetryTracerProvider((builder) => builder
+                    services.AddOpenTelemetryTracing((builder) => builder
```

Comment:
What if `AddOpenTelemetryTracing` is called multiple times?

Reply:
Addressing it separately. Current behavior is not good as it adds multiple tracerproviders, with newest one always overriding the previous ones, but leaving the old one *not* disposed.
This was reported separately. Will create an issue to track this.